s3torchconnectorclient/rust/src/mountpoint_s3_client.rs

```diff
@@ -42,26 +44,33 @@ pub struct MountpointS3Client {
     no_sign_request: bool,
     #[pyo3(get)]
     user_agent_prefix: String,
+    #[pyo3(get)]
+    endpoint: String,
 
     owner_pid: Pid,
 }
 
 #[pymethods]
 impl MountpointS3Client {
     #[new]
-    #[pyo3(signature = (region, user_agent_prefix="".to_string(), throughput_target_gbps=10.0, part_size=8*1024*1024, profile=None, no_sign_request=false))]
+    #[pyo3(signature = (region, user_agent_prefix="".to_string(), throughput_target_gbps=10.0, part_size=8*1024*1024, profile=None, no_sign_request=false, endpoint="".to_string()))]
     pub fn new_s3_client(
         region: String,
         user_agent_prefix: String,
         throughput_target_gbps: f64,
         part_size: usize,
         profile: Option<String>,
         no_sign_request: bool,
+        endpoint: String,
```

Comment:
Can we change this to be an `Option<String>` to be consistent with `profile`?

Reply:
Changed the type of endpoint parameter to Option<String>